### PR TITLE
Fsxac/proposal risk edits

### PR DESCRIPTION
**Pull request by FSXAC**
- Added risks
- Removed irrelevant risks
- Fixed grammar (as best as I could 🤔 )
- Fixed punctuation
- Re-evaluated risk likelihood, impact, and risk factor
- Sorted the risks again
- Added (&darr;) to indicate the table is sorted by risk factor in descending order
- Fixed decimal places so that all the numbers line up
- Renamed section title to "Potential Risks"

### CODE REVIEW

File: docs/proposal/3_8_risk_profile.tex [hunk 1]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
-Sabotage&0.1&1&0.1\\ \hline
-Loss of client&0.1&1&0.1\\ \hline
-Loss of team member&0.1&0.9&0.09\\ \hline
-Equipment shipment is delayed or lost.&0.1&0.8&0.08\\ \hline
-Client is not available enough to provide significant help.&0.1&0.5&0.05\\ \hline
-Competitive offerings affect project requirement&0.1&0.5&0.05\\ \hline
-Camera doesn’t work with FPGA&0.1&0.5&0.05\\ \hline
-FPGA doesn’t work with transmission systems&0.1&0.5&0.05\\ \hline
+
+Drone flight hardware (flight controllers, radio, motors) cannot function to crashes and damages. & 0.9 & 1.0 & 0.90 \\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Drone flight hardware (flight controllers, radio, motors) cannot function due to crash/damage. & 0.9 & 1.0 & 0.90 \\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 2]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Payload is too heavy which significantly increases drone motor requirements and significant reduction in flight duration. & 0.8 & 0.8 & 0.64 \\ \hline
+Accidents that damage the drone and computating equipment that require extra budget that we may not have. & 0.6 & 0.9 & 0.54\\ \hline
+Loss of drone hardware and payload during flight and cannot be recovered (total loss). & 0.5 & 1.0 & 0.50 \\\hline
+Not enough time commitment from team members. & 0.7 & 0.7 & 0.49\\ \hline
+Access to tools and shops for modifiying and repairing drone hardware is inadequate or non-existent. & 0.6 & 0.8 & 0.48\\ \hline
+Underestimation of project scope or work required, leading to insufficient time managemnet and burn-outs. & 0.5 & 0.9 & 0.45\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Underestimation of project scope or work required, leading to insufficient time management and burn-outs. & 0.5 & 0.9 & 0.45\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 3]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Payload is too heavy which exceeds total take-off weight. & 0.4 & 1.0 & 0.40 \\ \hline
+Legacy documents for the project is insufficient, resulting in the final product unable to be maintained or extended. & 0.7 & 0.5 & 0.35\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Legacy documents for the project are insufficient, resulting in poor maintainability/extensibility for the client. & 0.7 & 0.5 & 0.35\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 4]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Financial inefficiencies leading to overbudget or lack of capital. & 0.4 & 0.8 & 0.32\\ \hline
+Constrained to purchase lower-quality components due to budget and results in lower performance. & 0.6 & 0.5 & 0.30 \\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Constrained to purchase lower-quality components due to budget, resulting in lower performance. & 0.6 & 0.5 & 0.30 \\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 5]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Team is indecisive or team leader cannot make a timely decision --- results in delay. & 0.4 & 0.6 & 0.30\\ \hline
+Development and managment technique and methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Development and management technique/methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 6]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Not enough time to work on documentation. & 0.7 & 0.4 & 0.28\\ \hline
+Scope of the project is overestimated which leads to diversification of vision of the project. & 0.5 & 0.5 & 0.25\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Scope of the project is overestimated which leads to diversification of the project's vision. & 0.5 & 0.5 & 0.25\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 7]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Not enough machine learning training data. & 0.5 & 0.5 & 0.25\\ \hline
+Not enough FPGA logic elements to implement a desired ML model. & 0.5 & 0.5 & 0.25\\ \hline
+Failure to acquire regulatory compliance resulting in inability fly drone legally. & 0.3 & 0.7 & 0.21\\ \hline
+The software, tools or development environment for the project is inadequate. & 0.4 & 0.5 & 0.20\\ \hline
+Knowledge and skill regarding ML is insufficient. & 0.5 & 0.4 & 0.20 \\ \hline
+Technical debt paydown impact project timeline. & 0.4 & 0.5 & 0.20\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Technical debt paydown impacts project timeline. & 0.4 & 0.5 & 0.20\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 8]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Deliverables and objectives failing to meeting client’s expectations. & 0.2 & 0.9 & 0.18 \\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Deliverables and objectives fail to meet client’s expectations. & 0.2 & 0.9 & 0.18 \\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 9]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Client demands modification to the scope and requirements of the project that leads to delays or feature cuts. & 0.3 & 0.6 & 0.18\\ \hline
+Team lacking in effective communication which leads to overlapping work, missed work, and incompatible work. & 0.4 & 0.4 & 0.16\\ \hline
+Internal documentation or documentation for libraries and parts are not sufficient for development. & 0.4 & 0.4 & 0.16\\ \hline
+Camera module lacking in documentation. & 0.2 & 0.8 & 0.16\\ \hline
+New technology or research that changes the scope significantly. & 0.2 & 0.5 & 0.10\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
New technology or research emerges, changing the scope significantly. & 0.2 & 0.5 & 0.10\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 10]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Sabotage of the project. & 0.1 & 1 & 0.10\\ \hline
+Sudden loss of client. & $\leq$0.1 & 1 & 0.10\\ \hline
+Sudden loss of team member. & $\leq$0.1 & 0.9 & 0.09\\ \hline
+FPGA board lacks documentation. & $\leq$0.1 & 0.8 & 0.08\\ \hline
+Client is not cooperative or does not provide necessary information. & $\leq$0.1 & 0.8 & 0.08\\ \hline
+Key components are not available or takes too long to deliver. & 0.1 & 0.7 & 0.07\\ \hline
+Purchased orders of equipment or tools delayed or lost. & 0.1 & 0.5 & 0.05\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]: Same as one directly above?

File: docs/proposal/3_8_risk_profile.tex [hunk 11]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Client is not available enough to provide significant help. & $\leq$0.1 & 0.5 & 0.05\\ \hline
+Lacking resources in machine learning knowledge. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]: Already listed?

File: docs/proposal/3_8_risk_profile.tex [hunk 12]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Camera module fail to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Camera module fails to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 13]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Data transmitter fail to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Data transmitter fails to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex [hunk 14]
```diff
@@ -11,67 +11,75 @@ \subsubsection{Risk Management}
...
+Competitions on the market significantly affecting project requirement and scope. & $\leq$0.1 & 0.4 & 0.04\\ \hline
```
Comment (peterdeutsch) [inline docs/proposal/3_8_risk_profile.tex]:
```suggestion
Market competition significantly affects project requirements and scope. & $\leq$0.1 & 0.4 & 0.04\\ \hline
```

File: docs/proposal/3_8_risk_profile.tex (hunk 5 above)
Comment (FSXAC) [reply docs/proposal/3_8_risk_profile.tex]: What do you have against "and" or "or"s 😠

File: docs/proposal/3_8_risk_profile.tex (hunk 10 above)
Comment (FSXAC) [reply docs/proposal/3_8_risk_profile.tex]: Hmm, it appears that both were in there in the beginning but with different risk values, I rearranged and they're now together (which is a good thing I guess).

File: docs/proposal/3_8_risk_profile.tex (hunk 11 above)
Comment (FSXAC) [reply docs/proposal/3_8_risk_profile.tex]: One is lacking the knowledge to do something, the other is lacking the resource to acquire the knowledge. I'll make an edit to make this clearer

**Pull request merged**